cmd/truss/main.go

```diff
@@ -56,14 +55,18 @@ func main() {
 	genFiles, err := generateCode(cfg, dt, sd)
 	exitIfError(errors.Wrap(err, "cannot generate service"))
 
-	for _, f := range genFiles {
-		err := writeGenFile(f, cfg.ServicePath)
+	for path, file := range genFiles {
+		err := writeGenFile(file, path, cfg.ServicePath)
 		if err != nil {
 			exitIfError(errors.Wrap(err, "cannot to write output"))
 		}
 	}
 }
 
+// fileSystem is a map of paths relative to truss.Config.ServicePath mapped to
+// io.Reader's as files
```

Comment:
I thought that labeling it `fileSystem` would make functions that take in and return this value easier to read than:
```
func combineFiles(group ...fileSystem) fileSystem {
...
}

func combineFiles(group ...map[string]io.Reader) map[string]io.Reader {
...
}
```

If you disagree, then I have no attachment here.